packages/gatsby-theme-docs/src/layouts/content.js

```diff
@@ -38,16 +39,19 @@ const ResizableGrid = styled.div`
 const LayoutContent = props => {
   const [isMenuOpen, setMenuOpen] = React.useState(false);
   const data = useStaticQuery(graphql`
-    query GetTitle {
+    query GetSiteData {
       site {
+        pathPrefix
         siteMetadata {
           title
+          description
+          author
         }
       }
     }
   `);
   return (
-    <React.Fragment>
+    <SiteDataContext.Provider value={data.site}>
```

Comment:
We put the query result into a context, so that components can easily access it.